src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -224,6 +226,13 @@ func (gen *PythonStartupScriptGenerator) buildGunicornCommandForModule(module st
 	// Default to App Service's timeout value (in seconds)
 	args := "--timeout 600 --access-logfile '-' --error-logfile '-'"
 
+	// 2N+1 number of workers is recommended by Gunicorn docs.
```

Comment:
This is not what we want to do here. You can find more details in the work item's description. 
We had done a similar thing few months ago and then we noticed that it broke some users' apps mainly when they were using in-memory session scenarios.

Reply:
So as per yesterday's meeting, this code is good...we need to introduce a environment variable based flag - which if is true we set the worker count dynamically...Also let's first send out an email out to Jenny and Sanchit to come to an agreement on this.